mongo/bounties/final_validation.js

```diff
@@ -0,0 +1,73 @@
+db.createCollection("bounties", {
+  validator: {
+    $jsonSchema: {
+      bsonType: "object",
+      required: [
+        "season",
+        "title",
+        "description",
+        "criteria",
+        "reward",
+        "createdBy",
+        "createdAt",
+        "dueAt",
```

Comment:
I think status should also be required, as without it, we can not properly control permissions of API routes.